app/src/main/java/com/yacgroup/yacguide/network/SectorParser.kt

```diff
@@ -136,10 +137,11 @@ class SectorParser(private val _db: DatabaseWrapper) : JSONWebParser() {
         for (i in 0 until jsonComments.length()) {
             val jsonComment = jsonComments.getJSONObject(i)
             val user = jsonComment.getString("username")
+            val date = _parseCommentDate(jsonComment.getString("datum"))
```

Comment:
There is also an "adatum" field which may contain the date of the latest change. Would this make more sense?

Reply:
I had a look into the PHP code of [sandsteinklettern.de](sandsteinklettern.de) and for my understanding it is the archive date. It is updated, when the comment gets updated.

Reply:
I'm now prioritizing the `adatum` and use `datum` as fallback.